templates/zerver/auth_subdomain.html

```diff
@@ -0,0 +1,25 @@
+{% extends "zerver/portico.html" %}
+
+{% block portico_content %}
+
+<div class="error_page">
+    <div class="container">
+        <div class="row-fluid">
+            <img src="/static/images/500art.svg" alt=""/>
+            <div class="errorbox">
+                <div class="errorcontent">
+                    <h1 class="lead">Authentication subdomain</h1>
+                    <p>
+                        It appears you ended up here by accident. This site
+                        is meant to be an intermediate step in the authentication process
+                        and shouldn't be accessed manually. If you came here directly,
+                        you probably got the address wrong. If you got stuck here while trying
+                        to log in, this is most likely a server bug or misconfiguration.
```

Comment:
Could we just serve a redirect to /accounts/go rather than making this page?  I have some design concerns about it (E.g. it shows as a 500 even though it isn't, etc.), and I think it might be cleaner to just not have it exist.

Reply:
Hmm, what do you mean by ` it shows as a 500`? Just the visuals of the page that opens due to reusing the template we normally use for 500 pages? I figured it fit visually for what this page is supposed to convey, but we could use a very basic template like for invalid confirmation links as well. The actual status code of the response was 200 - I changed this to return `400` now because it seems more appropriate.

We could just return /accounts/go, but my reasoning for having a special page was about the kind of circumstances a user could possibly bump into the page - it generally should only happen if something goes wrong in the authentication flow, and that'll usually indicate a bug. So it doesn't seem helpful to the user to show them an "Find your accounts" page when the actual problem has nothing to do with it. A scenario where a user opens `auth.zulipchat.com` should be rarer - because why would they do that, `auth.zulipchat.com` doesn't show up in any user-facing places.

Reply:
Yeah, I think just showing the 500 error page template is the right model for something that's "Some sort of unexpected bug in the software", and avoids our having to design an additional page here.